Podfile

```diff
@@ -63,9 +68,12 @@ abstract_target 'defaults' do
   pod 'Cartography', '~> 3.1'
   pod 'GoogleUtilities', '~> 6.0'
 
-  target 'PactTests' do
+  pspdfkit
+
+  target 'Core' do
     project 'Core/Core.xcodeproj'
     pod 'PactConsumerSwift', :git => 'https://github.com/DiUS/pact-consumer-swift.git'
```

Comment:
PactConsumerSwift probably isn't needed for Core, just PactTests, right? This looks like a mistake.